.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -142,7 +142,7 @@ steps:
         WINDOWS_VERSION="$(cat vhdbuilder/packer/windows-image.env | grep -a "WINDOWS_2022_BASE_IMAGE_VERSION" | cut -d "=" -f 2 | cut -d "." -f 1,2)"
       else
         echo "Current distro is not supported to get image version. You need to update related code."
-        exit 1
+        WINDOWS_VERSION="unknown"
```

Comment:
What is the expected result with this change? It will fail the build before this change

Reply:
The artifact file will have "unknown" in the file name. https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=84214453&view=artifacts&pathAsName=false&type=publishedArtifacts

I want to add the support for 23H2 when the market place images are ready. Otherwise, I'm afraid that a fake 23H2 image version could make confusion.

Reply:
Added the code to support 23H2 here and keep "exit 1" for not supported distro.